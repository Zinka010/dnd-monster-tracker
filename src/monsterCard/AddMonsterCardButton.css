.add-monster-card-container {
    height: 400px;
    width: 350px;
    display: flex;
    justify-content: center;
    align-items: center;
}

.add-monster-card-plus-icon {
    width: 130px;
    height: 130px;
}

.add-monster-card {
    height: 175px;
    width: 175px;
    display: flex;
    justify-content: center;
    align-items: center;
    border: none;
    outline: none;
    background: rgba(0, 0, 0, 0.1);
    border-radius: 10px;
    color: rgba(0, 0, 0, 0.5);
    cursor: pointer;
    transition: background 0.2s;
}

.add-monster-card:hover {
    background-color: rgba(0, 0, 0, 0.2);
}
