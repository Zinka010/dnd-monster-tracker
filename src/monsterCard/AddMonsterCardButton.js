import React from "react";
import "./AddMonsterCardButton.css";
import AddFillIcon from "remixicon-react/AddFillIcon";

function AddMonsterCardButton({ handleClick }) {
  return (
    <div className="add-monster-card-container">
      <div className="add-monster-card" onClick={handleClick}>
        <AddFillIcon className="add-monster-card-plus-icon"></AddFillIcon>
      </div>
    </div>
  );
}

export default AddMonsterCardButton;
