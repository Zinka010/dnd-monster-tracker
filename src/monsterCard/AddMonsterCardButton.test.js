import renderer from 'react-test-renderer';
import AddMonsterCardButton from './AddMonsterCardButton';

it('assert render correctly', () => {
  const component = renderer.create(
    <AddMonsterCardButton handleClick={() => console.log("click")}/>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
