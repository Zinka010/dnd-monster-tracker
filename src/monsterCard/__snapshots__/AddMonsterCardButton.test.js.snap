// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`assert render correctly 1`] = `
<div
  className="add-monster-card-container"
>
  <div
    className="add-monster-card"
    onClick={[Function]}
  >
    <svg
      className="remixicon-icon add-monster-card-plus-icon"
      fill="currentColor"
      height={24}
      viewBox="0 0 24 24"
      width={24}
    >
      <path
        d="M11 11V5h2v6h6v2h-6v6h-2v-6H5v-2z"
      />
    </svg>
  </div>
</div>
`;
